homeassistant/const.py

```diff
@@ -360,6 +360,7 @@
 HTTP_HEADER_CACHE_CONTROL = 'Cache-Control'
 HTTP_HEADER_EXPIRES = 'Expires'
 HTTP_HEADER_ORIGIN = 'Origin'
+HTTP_HEADER_X_FORWARDED_FOR = 'X-Forwarded-For'
```

Comment:
Hi @balloob , I did a blame and this line was added some time ago:
```
e73634e6 (hexa-                2016-11-13 01:14:39 +0100 367) HTTP_HEADER_X_FORWARDED_FOR = 'X-Forwarded-For'
```

All other checks and conflicts are resolved.  I think it is good to go.  Let me know if not.

Thanks,
Whytey